tpm2/src/util/tpm2_kmyth_io.c

```diff
@@ -169,6 +169,523 @@ int read_arbitrary_file(char *input_path,
   return 0;
 }
 
+//############################################################################
+// tpm2_kmyth_create_ski_string
+//############################################################################
+int tpm2_kmyth_create_ski_string(uint8_t ** output,
```

Comment:
Looking at this function, the idea of having a SKI structure that can be written to file without an intermediate buffer seems really attractive.